tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Ods Organisations GetByOdsCode 3536.feature

```diff
@@ -0,0 +1,87 @@
+﻿Feature: Display an Ods Organisation by its Ods Code
+  As an Authority User
+  I want to view information for a single organisation
+  So that I can manage access to the Buying Catalogue
+
+Background:
+	Given Ods Organisations exist
+		| Name               | OdsCode | PrimaryRoleId | Status    | AddrLn1 | AddrLn2         | AddrLn3         | AddrLn4     | Town     | County          | Postcode | Country |
+		| Ods Organisation 1 | B1G     | Buyer         | Active    | 12      | Brick Lane      | Central Area    | City Centre | Leeds    | West Yorkshire  | LS1 1AW  | England |
+		| Ods Organisation 2 | B2G     | Buyer         | Disbanded | 37      | Sand Brick Lane | Central Area    | City Centre | Bradford | West Yorkshire  | BD4 1FK  | England |
+		| Ods Organisation 3 | B3G     | AlsoBuyer     | Active    | 21      | Jump Street     | All over        | Suburb      | Metairie | Louisiana       | 504      | USA     |
+		| Ods Organisation 4 | N0G     | 153XP         | Active    | 15      | Sun Ave         | End of the Road | Suburb      | York     | North Yorkshire | YO11 4LO | England |
+	And Organisations exist
+		| Name           | OdsCode |
+		| Organisation 1 | Ods 1   |
```

Comment:
Do you not want to relate the OdsCode here to one of the Ods before?

Reply:
I suppose I can 😄